server/src/nameQueries.js

```diff
@@ -0,0 +1,93 @@
+const knex = require('../knex.js');
```

Comment:
This module now has a more focused responsibility of interacting with the database only. It is not concerned with HTTP requests or responses from Express.

Reply:
I like how this approach separates the requests/response from the database interactions.  Seems like a good approach.